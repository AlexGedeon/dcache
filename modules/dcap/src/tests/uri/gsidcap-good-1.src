gsidcap://dcap-door:22223/path/to/file
