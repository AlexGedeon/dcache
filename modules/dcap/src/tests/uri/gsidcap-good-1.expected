URL: gsidcap://dcap-door:22223/path/to/file
  host: dcap-door:22223
  file: path/to/file
  type: 1
  prefix: gsi
  config line: dcap-door:22223:libgsiTunnel.so
